examples/advanced/variables.tf

```diff
@@ -16,6 +16,17 @@ variable "ibmcloud_api_key" {
   sensitive   = true
 }
 
+variable "provider_visibility" {
```

Comment:
If we know the example only works on private due to the CBR rules being added, you should not expose the provider visibility in the example - otherwise someone can change it to pubic and it will fail.